### DISCUSSION TITLE
Dividing a list of orders by categories to get more efficiency

### DISCUSSION THREAD

**Discussion by 20Artee** [Steffo99/greed#184] [Q&A] [answered]
Hello everyone. I would like to listen to advice about orders, I want to see in the order different categories in which there will be a list of services, but in the implementation it is difficult for me to edit the finished code.For example, I have 110 different services, I would like to divide them into different blocks so that you can choose say 5 different categories each of 22, is it feasible with this project?

**Comment by Steffo99** [comment, marked as answer]
It has been requested many times (#33), and a development branch for the feature was started as well (#88), but it never got anywhere since the main author disappeared and without a database migrations framework (#74) it would break backwards compatibility with older versions of the bot.

I think it's definitely feasible, but I'm not personally interested in developing it.

I'd gladly take a look to high quality pull requests solving these problems, so please consider developing it yourself :)

**Reply by Ebsbrown** [reply]
I’ll pay for this feature if you’re interested @Steffo99

**Reply by Steffo99** [reply]
I've already said that I'm not interested in developing it, I'm sorry @Ebsbrown...
